eva_submission/xlsx/xlsx_parser_eva.py

```diff
@@ -18,13 +18,13 @@
 from cached_property import cached_property
 from ebi_eva_common_pyutils.logger import AppLogger
 
-from eva_submission.xls_parser import XLSReader, XLSWriter
+from eva_submission.xlsx.xlsx_parser import XLSReader, XLSWriter
 
 
 class EVAXLSReader(AppLogger):
 
     def __init__(self, metadata_file):
-        conf = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'etc', 'eva_project_conf.yaml')
+        conf = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'etc', 'eva_project_conf.yaml')
```

Comment:
i might suggest having a constant somewhere called something like `PROJECT_ROOT` (probably not a good choice since project is used for other things in this context, but you know what i mean), which you can reuse anytime you need get the path of something else in the entire python project.  this would just make any lines of code like this a bit more readable.

something like this perhaps: https://stackoverflow.com/a/25389715

Reply:
That can work for the main module but not for the executables. I've changed it for the modules